### PR TITLE
Fix more warnings

### PR DESCRIPTION
**Pull request by cbrnr**
I've added smoke tests for XDF test files in a local folder. I found two warnings in `_clock_sync`:

- [x] If `clock_times` (and `clock_values`) is a list with only one element, `time_diff` is an empty list and `np.median` of an empty list raises a warning. I've solved this by handling clock resets only for two or more time stamps. I hope this change doesn't affect anything else I might have overlooked.
- [x] With another test file, I get a warning in line 633:
    ```
    RuntimeWarning: divide by zero encountered in true_divide
    tmp = np.maximum(0, (1 - (1 + 1 / rho) / np.abs(d)))
    ```
    This is because `d` contains elements that are exactly 0. I have disabled this warning for this particular line.

### CODE REVIEW

**Comment by cbrnr** [comment]
@tstenner @cboulay ready to merge - please double-check if these changes have any unintended side effects.

**Comment by cbrnr** [comment]
Also, the test I've added requires a folder with XDF files. We can leave this test in for now, but we should start discussing how we could host test files. For most of the files I have, I will get permission from the original owners to use them in our tests.

**Comment by cbrnr** [comment]
git-lfs should work with CIs even when a repo is cloned before git-lfs is installed. But let's continue this in #38. Meanwhile, I have removed the test again so that this PR can get merged.

**Comment by tstenner** [comment]
We could put this example file in the repo:

[test.xdf.gz](https://github.com/xdf-modules/xdf-python/files/3711592/test.xdf.gz)

Its size is somewhat reasonable for a repository (554 bytes) and it has two streams, so for a quick check it should suffice.

**Comment by cbrnr** [comment]
Let's talk about test data in #38. This PR is ready to be merged, could someone please review?

File: pyxdf/pyxdf.py [hunk 1]
```diff
@@ -630,7 +630,8 @@ def _robust_fit(A, y, rho=1, iters=1000):
     for k in range(iters):
         x = np.linalg.solve(U, (np.linalg.solve(L, Aty + np.dot(A.T, z - u))))
         d = np.dot(A, x) - y + u
-        tmp = np.maximum(0, (1 - (1 + 1 / rho) / np.abs(d)))
+        with np.errstate(divide="ignore"):  # ignore divide by zero warning
```
[the remark this answers is not in the document]
Comment (cbrnr) [reply pyxdf/pyxdf.py]: Neat - didn't know about the `where` parameter. Done.

[the remark this answers is not in the document]
Comment (tstenner) [reply pyxdf/pyxdf.py]: I also learned about it an hour ago

**Pull request merged**